vehicle/swagger.yaml

```diff
@@ -70,7 +82,7 @@ paths:
           description: "Organization authenticity code."
       responses:
         200:
-          description: "Successfully returned uvin and its associated vehicle information."
+          description: "Successful"
           schema:
             $ref: '#/definitions/VehicleReg'
         401:
```

Comment:
Need a description on 404 due to potential confusion with empty 200 responses.

Reply:
yes I agree -   will get the explanation in the next version